### PR TITLE
Fix all WebRTC_API links

### PR DESCRIPTION
**Pull request by AnilSeervi**
#### Summary

- Fixes all WebRTC_API links
#### Motivation

open-source
#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/audiotracklist/addtrack_event/index.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ---
-title: 'AudioTrackList: addtrack event'
+title: "AudioTrackList: addtrack event"
```
Comment (wbamberg) [inline files/en-us/web/api/audiotracklist/addtrack_event/index.md:2]:
```suggestion
title: 'AudioTrackList: addtrack event'
```

Comment (wbamberg) [inline files/en-us/web/api/audiotracklist/addtrack_event/index.md:2]:
```suggestion
title: 'AudioTrackList: addtrack event'
```

File: files/en-us/web/api/audiotracklist/removetrack_event/index.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
 ---
-title: 'AudioTrackList: removetrack event'
+title: "AudioTrackList: removetrack event"
```
Comment (wbamberg) [inline files/en-us/web/api/audiotracklist/removetrack_event/index.md:2]:
```suggestion
title: 'AudioTrackList: removetrack event'
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for this PR @AnilSeervi !

It looks like this has Prettified the files involved (or used some other opinionated formatter) - as a rule I'd prefer a PR not to do that, and just to make the fix it wants to. But it's OK, and the Prettier version is much better in a few places, so I'm mostly happy with it.

The only bits I complained about are where it's using double-quotes for front matter strings - AFAIK we consistently use single-quotes here, so it seems like it will be confusing to change it in some random cases. One day we will run Prettier over everything and things will be better.

**Comment by AnilSeervi** [comment]
I did this change in **codespaces** and as there is no prettier config file in the repo it looks like the config I use was enforced even tho what I did was search and replace every instance.

And because I did it in the web editor, it auto-saved the files that I searched and replaced again, triggering prettier-at-save.

A container can be set-up for working with codespaces like [github/docs ](https://github.com/github/docs) repo has.
More about how to setup here - https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/configuring-codespaces-for-your-project

**Review by wbamberg — APPROVED** [review]
👍 thank you @AnilSeervi :)

**Pull request merged**